packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -0,0 +1,75 @@
+import 'dart:async';
+
+// ignore: implementation_imports
+import 'package:test/src/executable.dart' as test;
+
+import '../artifacts.dart';
+import '../base/common.dart';
+import '../base/file_system.dart';
+import '../base/io.dart';
+import '../base/terminal.dart';
+import '../dart/package_map.dart';
+import '../globals.dart';
+import '../test/flutter_platform.dart' as loader;
+import 'watcher.dart';
+
+/// Runs tests using package:test and the Flutter engine.
+Future<int> runTests(List<String> testFiles, {Directory workDir,
+  bool enableObservatory: false, bool startPaused: false, bool ipv6: false,
+  TestWatcher watcher}
+  ) async {
+
+  // Compute the command-line arguments for package:test.
+  final List<String> testArgs = <String>[];
+  if (!terminal.supportsColor) {
+    testArgs.addAll(<String>['--no-color', '-rexpanded']);
+  }
+  if (enableObservatory != null) {
+    testArgs.add('--concurrency=1');
```

Comment:
This was originally added for coverage only, and I wanna make sure we're not conflating it with `enableObservatory` -- do we need this if `startPaused` is true?